tests/monotouch-test/ModelIO/MDLTexture.cs

```diff
@@ -41,6 +40,7 @@ public void Setup ()
 		{
 			TestRuntime.AssertXcodeVersion (7, 0);
 
+#if !MONOMAC
```

Comment:
The bug this section is about is also present in i386 on mac, so instead of #ifdefing the whole section, you should only ifdef the simulator check.